src/index.js

```diff
@@ -92,7 +93,14 @@ class Game extends React.Component {
       const desc = move ? "Go to move #" + move : "Go to game start";
       return (
         <li key={move}>
-          <button onClick={() => this.jumpTo(move)}>{desc}</button>
+          <button onClick={() => this.jumpTo(move)}>{desc}</button>{" "}
```

Comment:
This is a test comment. Please resolve it. Thanks!

Reply:
Test comment observed! Thank YOU